docs/index.md

```diff
@@ -38,18 +35,20 @@ However, you don't want to run this set of xarray operations every time you open
 
 What's being cached here, you ask? We're effectively caching the result of performing all the various consistency checks that xarray performs when it combines newly-encountered datasets together. Once you have the new virtual Zarr store xarray is able to assume that this checking has already been done, and trusts your Zarr store enough to just open it instantly.
 
+### Usage
+
 Creating the virtual store looks very similar to how we normally open data with xarray:
 
 ```python
-import virtualizarr  # required for the xarray backend and accessor to be present
+from virtualizarr import open_virtual_dataset
 
-virtual_ds = xr.open_mfdataset(
-    '/my/files*.nc',
-    engine='virtualizarr',  # virtualizarr registers an xarray IO backend that returns ManifestArray objects
-    combine='by_coords',  # 'by_coords' stills requires actually reading coordinate data
-)
+virtual_datasets = [
+    open_virtual_dataset(filepath)
+    for filepath in glob.glob('/my/files*.nc')
+]
```

Comment:
I changed this aspirational example in the Readme to one that's not quite as neat, but should actually work right now.